cmd/config.go

```diff
@@ -57,14 +66,24 @@ func initializeConfig(cmd *cobra.Command, bindPrefix string) error {
 		// Check the base path (e.g., package-name)
 		configPaths = append(configPaths, f.Name)
 		
+		logger.Default.Debug(fmt.Sprintf("Looking for config value for flag %s in paths: %s", f.Name, strings.Join(configPaths, ", ")))
+		
 		// Try each path in order until we find a match
 		for _, path := range configPaths {
 			if v.IsSet(path) {
 				val := v.Get(path)
-				_ = f.Value.Set(fmt.Sprintf("%v", val))
-				break
+				err := f.Value.Set(fmt.Sprintf("%v", val))
+				if err != nil {
```

Comment:
You could consider using error level here, up to you.

Reply:
Thanks for the feedback but I'd prefer to leave it as is for now.